lib/WorseReflection/Core/Type/IntLiteralType.php

```diff
@@ -18,10 +18,8 @@ public function __toString(): string
         return (string)$this->value;
     }
 
-    /**
-     * @return int
-     */
-    public function value()
+
+    public function value(): int
```

Comment:
i'm confused by both of htese changes can child classes narrow the type proided in an interface (both of these methods come from the interface)

Reply:
hmm, ok, since 7,4 and co-variance this is fine :shrug: https://wiki.php.net/rfc/covariant-returns-and-contravariant-parameters